examples/android/android-studio/Cukeulator/README.md

```diff
@@ -0,0 +1,72 @@
+## Cukeulator Example Test
+This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
```

Comment:
Sure, I've updated Android Studio to 1.0.2 and also updated the project and
readme. I had to make a minor change to build.gradle - runProguard property
has changed to minifyEnabled (
http://tools.android.com/tech-docs/new-build-system/migrating-to-1-0-0).

On Mon, Jan 5, 2015 at 12:53 AM, Sebastian Gröbler <notifications@github.com
> wrote:

> In examples/android/android-studio/Cukeulator/README.md
> <https://github.com/cucumber/cucumber-jvm/pull/816#discussion-diff-22451236>
> :
>
> > @@ -0,0 +1,72 @@
> > +## Cukeulator Example Test
> > +This is the example test-project for the Cukeulator app for Android Studio (Beta) 0.8.14.
>
> I think we can change this to "Android Studio 1.0.1"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cucumber/cucumber-jvm/pull/816/files#r22451236>.
>